mbon_data_uploader <pre>v{{error_dict.__version__}}</pre>
<br>
<br>
HTTPException <pre>{{error_dict.code}}</pre>
<br>
<br>
Description: <br>
{{error_dict.description}}
<br>
<br>
Stack trace: <br>
<pre>{{error_dict.stack_trace}}</pre>
<br>
<br>
Subprocess error details:<br>
command: {{error_dict['cmd']}}
<br>
<br>
return code: <pre>{{error_dict[returncode]}}</pre>
<br>
<br>
output: <br>
<pre>{{error_dict['output']}}</pre>
<br>
<br>
stdout: <br>
<pre>{{error_dict['stdout']}}</pre>
<br>
<br>
stderr: <br>
<pre>{{error_dict['stderr']}}</pre>
<br>
<br>

To report this error please open a "new issue" in
<a href="https://github.com/marinebon/mbon_data_uploader/issues">
    the code repository's issue tracker
</a>
and include the details above.
